user-guide/settings/gateway.md

```diff
@@ -8,33 +8,43 @@ position: 30
 
 # Gateway Menu
 
-The __Gateway__ sub-menu provides options to easily configure how Fiddler Everywhere accesses the network. By default, Fiddler Everywhere "chains" to the system's default proxy. These settings allows you to overwrite that behavior.
+The __Gateway__ sub-menu provides options to easily configure how Fiddler Everywhere accesses the network. By default, Fiddler Everywhere "chains" to the system's default proxy. These settings allow you to overwrite that behavior.
 
 ![Example manual proxy configuration](../../images/settings/gateway-system-proxy.png)
 
 ## Use system proxy
 
-__Use system proxy (recommended)__ is the default selection. Fiddler uses the OS system proxy. Some browsers and many applications use the system proxy by default and re notified when it changes. 
+__Use system proxy (recommended)__ is the default selection. Fiddler uses the OS system proxy. Some browsers and many applications use the system proxy by default and are notified when it changes. 
 
 ## Manual proxy configuration
 
 Select __Manual proxy configuration__  to specify the upstream proxy string (first textbox) and proxy bypass list (second textbox). 
 
-__Proxy configuration string__
+__Proxy string__
```

Comment:
When you describe menus or options, try to use the same naming as the GUI. For example, use _Proxy string_ instead of _Proxy configuration string_. This could also be a leftover from a previous Fiddler version.